staging/src/k8s.io/apiserver/pkg/storage/etcd3/metrics/metrics.go

```diff
@@ -86,15 +95,17 @@ func Register() {
 	registerMetrics.Do(func() {
 		legacyregistry.MustRegister(etcdRequestLatency)
 		legacyregistry.MustRegister(objectCounts)
-		legacyregistry.MustRegister(dbTotalSize)
+		legacyregistry.MustRegister(etcdObjectCounts)
+		etcdObjectCountstry.MustRegister(dbTotalSize)
```

Comment:
typo: should be etcdObjectCounts

Reply:
ou actually this should be legacyregistry

Reply:
Fixed and thanks! 

I just noticed the same thing from the failing tests... No idea why it got in.